pkg/nsx/cluster.go

```diff
@@ -229,9 +229,6 @@ func (cluster *Cluster) Health() ClusterHealth {
 }
 
 func (cluster *Cluster) GetVersion() (*NsxVersion, error) {
-	if len(nsxVersion.NodeVersion) > 0 {
```

Comment:
Remove local cached NodVersion  and read value by API request each time. This is to avoid cached value not changed even nsx manger is upgraded to new version.